src/appBootstrapper.jsx

```diff
@@ -43,6 +43,49 @@ initializeConfiguration();
 // THIS TELLS WEBPACK BUNDLE LOADER WHERE TO LOAD SPLIT BUNDLES
 __webpack_public_path__ = AppConfig.frontendUrl;
 
+// TODO: temparary fix the wheel event described in: https://github.com/facebook/react/issues/14856
```

Comment:
lets put this in a imported function from some util file.  handleTouchEventsForChromeBug or something like that.  just don't want this file getting too crowded

Reply:
Sure!